app/routes/events/view/speakers/list.js

```diff
@@ -10,6 +10,12 @@ export default class extends Route.extend(EmberTableRouteMixin) {
         return this.l10n.t('Accepted');
       case 'rejected':
         return this.l10n.t('Rejected');
+      case 'confirmed':
+        return this.l10n.t('Confirmed');
+      case 'withdrawn':
+        return this.l10n.t('Withdrawn');
+      case 'canceled':
+        return this.l10n.t('Canceled');
```

Comment:
Please use this - 
```
import { capitalize } from 'lodash-es';
titleToken() {
    if (['accepted', 'rejected', 'confirmed', 'withdrawn', 'canceled'].includes(this.params.speakers_status)) {
      return capitalize(this.params.invoice_status);
    } else {
      return this.l10n.t('All');
    }
  }

```
for more info - 
https://github.com/fossasia/open-event-frontend/blob/5c50d993703c8aef1a043ae92b4a469e2e43c894/app/routes/account/billing/invoices/list.js

Reply:
Wrap capitalize in tVar